app/src/full/kotlin/io/homeassistant/companion/android/settings/wear/views/SettingsWearOnboardingView.kt

```diff
@@ -0,0 +1,165 @@
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.PreviewLightDark
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import io.homeassistant.companion.android.common.R as commonR
+import io.homeassistant.companion.android.settings.wear.SettingsWearViewModel
+import timber.log.Timber
+
+@Composable
+fun SettingsWearOnboardingView(
+    settingsWearViewModel: SettingsWearViewModel,
+    onInstallOnWearDeviceClicked: () -> Unit,
+    onFinishInstallOnDevices: () -> Unit,
+    onBackClicked: () -> Unit,
+) {
+    var infoTextResource by remember { mutableIntStateOf(0) }
+    var shouldDisplayRemoteAppInstallButton by remember { mutableStateOf(false) }
+    val wearNodesWithApp by settingsWearViewModel.wearNodesWithApp.collectAsStateWithLifecycle()
+    val allConnectedNodes by settingsWearViewModel.allConnectedNodes.collectAsStateWithLifecycle()
+    when {
+        wearNodesWithApp == null || allConnectedNodes == null -> {
+            Timber.d("Waiting on Results for both connected nodes and nodes with app")
+            infoTextResource = commonR.string.message_checking
+            shouldDisplayRemoteAppInstallButton = true
+        }
+
+        allConnectedNodes?.isEmpty() == true -> {
+            Timber.d("No devices")
+            infoTextResource = commonR.string.message_no_connected_nodes
+            shouldDisplayRemoteAppInstallButton = true
+        }
+
+        wearNodesWithApp?.isEmpty() == true -> {
+            Timber.d("Missing on all devices")
+            infoTextResource = commonR.string.message_missing_all
+            shouldDisplayRemoteAppInstallButton = true
+        }
+
+        (wearNodesWithApp?.size ?: 0) < (allConnectedNodes?.size ?: 0) -> {
+            Timber.d("Installed on some devices")
+            onFinishInstallOnDevices.invoke()
+        }
+
+        else -> {
+            Timber.d("Installed on all devices")
+            onFinishInstallOnDevices.invoke()
+        }
+    }
+
+    SettingsWearOnboardingViewContent(
+        infoTextResource,
+        shouldDisplayRemoteAppInstallButton,
+        onInstallOnWearDeviceClicked,
+        onBackClicked,
+    )
+}
+
+@Composable
+fun SettingsWearOnboardingViewContent(
+    infoTextTitleResource: Int,
+    shouldDisplayRemoteAppInstallButton: Boolean,
+    onInstallOnWearDeviceClicked: () -> Unit,
+    onBackClicked: () -> Unit,
+) {
+    Scaffold(
+        topBar = {
+            SettingsWearTopAppBar(
+                title = {
+                    Text(
+                        stringResource(commonR.string.wear_os_settings_title),
+                        fontWeight = FontWeight.Bold,
```

Comment:
The other screen using the SettingsWearTopAppBar are not setting the frontWeight did you compare how it looks?